src/main/java/de/thomas_oster/visicut/gui/MainView.java

```diff
@@ -2058,18 +2077,17 @@ private String getJobName()
     String jobname = "(unnamed job)";
     try
     {
+      // CAUTION: keep in sync with job name construction below in run()
```

Comment:
Is it possible to deduplicate the code? Then we could show the final job name (and not just the prefix) in the textbox. In my opinion, the label "Job name prefix" doesn't sound very user-friendly.

Reply:
Not sure about the deduplication. Could break the threadding? or not?

the final job name has some parts that are not user editable. This can be an autoincremented Job Number or the full file name at the end. Depending on a users prefernce setting.

Showing the final job name next or below the prefix would be great. I don't dare to touch the XML for changing the GUI.

Reply:
You can change the UI from within Netbeans UI editor quite comfortable, you just need to have two plugins installed https://github.com/t-oster/VisiCut/wiki/Development:-Getting-started#editing-the-graphical-user-interface-layout